### PR TITLE
Feat/profile

### PR DESCRIPTION
**Pull request by EmanMohamed223**
### 📄 Description
profile screen done 
 - get all data of the user logged in 
 - user can update hie/her data

### CODE REVIEW

File: KlivvrSocialAppIOS/KlivvrSocialAppIOS/KlivvrFoundation/Externals  /DataNetwork/NetworkManager.swift [hunk 1]
```diff
@@ -57,9 +57,11 @@ class AlamofireManager: NetworkProtocol {
         } else {
             AF.request(endpoint).responseJSON { response in
                 do {
+                    print(response.response?.statusCode)
```
Comment (iramzyy) [inline KlivvrSocialAppIOS/KlivvrSocialAppIOS/KlivvrFoundation/Externals  /DataNetwork/NetworkManager.swift:60]: Remove print statement

File: KlivvrSocialAppIOS/KlivvrSocialAppIOS/KlivvrFoundation/Externals  /DataNetwork/NetworkManager.swift [hunk 2]
```diff
@@ -57,9 +57,11 @@ class AlamofireManager: NetworkProtocol {
...
                     guard let statusCode = response.response?.statusCode else { return }
                     switch statusCode {
                     case 200:
+                        print(response.value)
```
Comment (iramzyy) [inline KlivvrSocialAppIOS/KlivvrSocialAppIOS/KlivvrFoundation/Externals  /DataNetwork/NetworkManager.swift:64]: Remove print statement

**Review by dalia61 — APPROVED** [review]

File: KlivvrSocialAppIOS/KlivvrSocialAppIOS/Modules/Profile/Presenter/ProfilePresenter.swift [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        self.presenterOutput = presenterOutput
+    }
+}
+
+// MARK: ProfileInputPresenter
+
+extension ProfilePresenter: ProfileInputProtocol {
+    func updateInterests(interest: String) {
+        updateEnabledStateButton()
+        if interest == "" {
```
Comment (iramzyy) [inline KlivvrSocialAppIOS/KlivvrSocialAppIOS/Modules/Profile/Presenter/ProfilePresenter.swift:31]: replace it with if interest != "" {
// do your thing
}

File: KlivvrSocialAppIOS/KlivvrSocialAppIOS/Modules/Profile/Presenter/ProfilePresenter.swift [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+                                          mimeType: "image/jpg", name: "image")
+        self.multipartData = multipartData
+        updateEnabledStateButton()
+    }
+}
+// MARK: Private Handlers
+
+private extension ProfilePresenter {
+    func updateEnabledStateButton() {
+//        let isPhoneNumberValid = !phoneNumber.isEmpty
```
Comment (iramzyy) [inline KlivvrSocialAppIOS/KlivvrSocialAppIOS/Modules/Profile/Presenter/ProfilePresenter.swift:97]: Remove commented code

File: KlivvrSocialAppIOS/KlivvrSocialAppIOS/Modules/Profile/ProfileEndPoint.swift [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                headers.add(name: "authorization", value: "Bearer \(token)")
+            return headers
+        }
+    }
+
+    var parameters: Parameters? {
+        let param = defaultParams
+        switch self {
+        case let .update(phone,interests,aboutMe, address):
+          //  var endPointParam = [:]
```
Comment (iramzyy) [inline KlivvrSocialAppIOS/KlivvrSocialAppIOS/Modules/Profile/ProfileEndPoint.swift:40]: remove commented code

File: KlivvrSocialAppIOS/KlivvrSocialAppIOS/Modules/Profile/View/ProfileViewController.swift [hunk 6]
```diff
@@ -8,21 +8,129 @@
...
+        } else if UIImagePickerController.isSourceTypeAvailable(.photoLibrary) {
+            picker.sourceType = .photoLibrary
+            picker.allowsEditing = true
+        }
+        self.present(picker, animated: true)
+    }
+func imagePickerController(_ picker: UIImagePickerController,
+                           didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey: Any]) {
+        if let image = info[UIImagePickerController.InfoKey.originalImage] as? UIImage {
+          //  addImageButton.isHidden = true
```
Comment (iramzyy) [inline KlivvrSocialAppIOS/KlivvrSocialAppIOS/Modules/Profile/View/ProfileViewController.swift:107]: remove commented code

**Comment by dalia61** [comment]
feat:profile screen

**Pull request merged**